tests/lcs/core/graphics/graphics.livecodescript

```diff
@@ -250,3 +250,133 @@ on TestTeardown
 	end if
 end TestTeardown
 
+
+command _TestGraphicsPointOverflowNoThrow
+	local tPoints
+
+	repeat with i = 1 to 256
+		repeat with j = 1 to 256
+			if i is 256 and j is 256 then
+				exit repeat 
+				exit repeat
+			end if
+			put (i,j) & cr after tPoints
+		end repeat
+	end repeat
+
+	TestAssert "the number of gen points is correct", the number of lines in tPoints is 65535
+
+	create graphic
+	set the points of it to tPoints
+	set the markerPoints of it to tPoints
+end _TestGraphicsPointOverflowNoThrow
+
+on TestGraphicPointOverflow
+	local tE1, tE2, tE3, tE4, tE5, tPoints
+
+	repeat with i = 1 to 256
+		repeat with j = 1 to 256
+			put (i,j) & cr after tPoints
+		end repeat
+	end repeat
+
+	TestAssert "generated correct number of points", the number of lines in tPoints is 65536
+
+	try
+		create graphic 
+		set the points of it to tPoints
+	catch tError
+		put tError into tE1
+	end try
+
+	TestAssert "throws (points)", item 1 of line 1 of tE1 is "0896"
```

Comment:
These error assertions should be using `TestAssertThrow`, i.e.
```
on _TestSetGraphicPointsThrow pPoints
   create graphic
   set the points of it to pPoints
end _TestSetGraphicPointsThrow

on TestGraphicPointOverflow
...
TestAssertThrow "throw when setting number of graphic points to 65536", _TestSetGraphicPointsThrow, \
   the long id of me, "EE_GRAPHIC_TOOMANYPOINTS", tPoints

Note the API has changed so that you use the 'tag' of the error enum rather than the number.